.github/workflows/pr-test-rust.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           pip install --force-reinstall dist/*.whl
       - name: Run e2e test
         run: |
-          bash scripts/killall_sglang.sh
+          bash scripts/killall_sglang.sh "nuk_gpus"
```

Comment:
Why does pr-test-rust need this parameter?

Reply:
Seems like most of the time GPU memory isn't cleaned up properly during those tests; so it's to make sure we can load the model for tests.